Add post on jabref's bday (#65)
JabRef/blog.jabref.org: _posts/2022-12-06_Jabref-Birthday-Nikolaus.md

---
title:  JabRef's 19th Birthday
id: JabRefBirthday22
author: "[JabRef Maintainers](https://github.com/JabRef/jabref/blob/main/MAINTAINERS)"
bg: jabref-font
color: white 
---

> Last week, on November 29, it was JabRef's 19 birthday! Yep. That old already 😉. While the original creators Morten O. Alver and Nizar Batada are no longer active developing at JabRef, they are the ones who led the foundation in 2003 for the basis of JabRef. And even though they are no longer on board, we have a couple of users who still use JabRef since the early days!

Reviewer: ```suggestion
Last week, on November 29, it was JabRef's 19 birthday! Yep. That old already 😉. While the original creators Morten O. Alver and Nizar Batada are no longer actively developing JabRef, they led the foundations in 2003. Moreover, we have a couple of users who use JabRef since the early days!
```

Over the years JabRef has seen many contributors coming and going, and we would like to thank all of them for their various contributions. This not only includes code but also ideas and translations!

![The JabRef Logo with colored balloons](../img/jabref%2019%20years.png){: style="float:right"}

Fast-forward to December 2022, we are planning the release of Version 5.8 around the upcoming holidays. Now is a good point to test the latest development version and report any issues you encounter.

We are excited about the upcoming release and the cool new features! 😎

> ## Special Thanks

Reviewer: I would move this up before the picture.

We want to thank all the users who are constantly testing the latest snapshots and providing feedback!

> Furthermore, we would like to thank our [donators](https://donations.jabref.org) and JabRef e.V. members who made it possible for many JabRef developers to attend the conference and to support our work.

Reviewer: which conference?
